### PR TITLE
statuses

### PR DESCRIPTION
**Pull request by callumjt**
adds changing statuses 

ok scream at me for my code now red :3

### CODE REVIEW

File: modules/reminders/send.ts [hunk 1]
```diff
@@ -194,6 +195,37 @@ async function sendReminders(): Promise<NodeJS.Timeout | undefined> {
...
+					let status = [
+						"Watching the SA server!",
+						"Watching for bugs...",
+						"Dating Callum",
+						"Yes",
+						"Auto-Moderating the SA Server!",
+						"Farming dangos",
+						"Rico, status",
+						"Scanning potatoes",
+					];
```
Comment (RedGuy12) [inline modules/reminders/send.ts:208]:
```suggestion
					const statuses = [
						"Watching the SA server!",
						"Hope for no bugs…",
						"Dating Callum",
						"e",
						"Moderating Scratch Addons",
						"Hi, I’m Scradd!",
						"Rico, status",
						"Scanning potatoes",
						"Try /addon!",
						"beep boop beep",
						"ims scradd",
						"alan 👑",
						"strawberries 😌",
					].toSorted(() => Math.random() - 0.5);
```

Comment (RedGuy12) [reply modules/reminders/send.ts:208]: also rename the two other times you use `status`

File: modules/reminders/send.ts [hunk 2]
```diff
@@ -194,6 +195,37 @@ async function sendReminders(): Promise<NodeJS.Timeout | undefined> {
...
+					const next = (Number(reminder.reminder) + 1) % status.length;
+
+					remindersDatabase.data = [
+						...remindersDatabase.data,
+						{
+							channel: "0",
+							date: Number(Date.now() + 18_000_000),
```
Comment (RedGuy12) [inline modules/reminders/send.ts:220]:
```suggestion
							date: Date.now() + ((Math.random() * 3) + 3) * 3_600_000,
```
to randomly change every 3-6 hours (i changed my mind again)
(i like randomness)

Comment (callumjt) [reply modules/reminders/send.ts:220]: lol

**Pull request merged**